pages/common/xzless.md

```diff
@@ -0,0 +1,9 @@
+# xzless
+
+> Filters and displays text from `.xz` and `.lzma` files.
+> Uses `less` to present the output.
```

Comment:
Perhaps this better
```suggestion
> See also: `less`.
```

Reply:
I wanted to say "It uses the `less` output format to display the contents". Since the suggestion looks fine I'll commit it.